spacy/ml/models/coref_util.py

```diff
@@ -201,3 +203,17 @@ def create_gold_scores(
 
     # caller needs to convert to array, and add placeholder
     return out
+
+
+def spans2ints(doc):
```

Comment:
To follow the naming conventions in the IOB utils and other conversions I'd suggest renaming this to at least `spans_to_offsets`.

It's unfortunate that we use `offsets` for both token and character offsets, but `spans_to_char_offsets` gets kind of clunky, or even clunkier `unlabeled_spans_to_char_offsets` so it's not confused with the utilities for labeled spans.

Maybe it should just be private, especially if it's effectively only a test util?

Reply:
Thanks, I'll rename it to `_spans_to_offsets` - originally this was just in one of the tests, but I moved it to util because it was relevant for both components. 

I think we do need a util function that can say "these SpanGroups from different docs are equivalent" in a robust way, but that isn't needed for these tests / in this PR. (Maybe there's already something like that for spancat?)